editions/tw5.com/tiddlers/webserver/WebServer Header Authentication.tid

```diff
@@ -8,6 +8,8 @@ Header authentication is a web integration technique enabling external entities
 
 Header authentication is activated if is configured via the [[authenticated-user-header|WebServer Parameter: authenticated-user-header]]
 
+Here is an [example guide](https://www.pomerium.com/docs/guides/tiddlywiki) of how to add authentication and authorization to a TiddlyWiki instance with Pomerium, an open-source proxy.
```

Comment:
The link is markdown and will not work in TW. It needs to be `[[example guide|https://www.pomerium.com/docs/guides/tiddlywiki]]`

Reply:
I think this paragraph should be at the end of the tiddler under a new heading eg:

```
!! 3rd Party Projects That Support This Mechanism
```

or something similar.